corehq/apps/domain/templates/login_and_password/two_factor/core/login.html

```diff
@@ -5,6 +5,14 @@ <h2 class="text-center">
     {{ current_page.page_name }}
   </h2>
 
+  {% if messages %}
```

Comment:
This block is a simplified version of [this one](https://github.com/dimagi/commcare-hq/blob/f124be6b14f7efdb9186b34c3265293752d49e2f/corehq/apps/hqwebapp/templates/hqwebapp/base.html#L257-L282).